packages/core/src/fil/depositAddress.ts

```diff
@@ -1,31 +1,52 @@
 import BN from "bn.js";
 
 import {
-  FilWalletData,
   FilAbstractWallet,
   FilAbstractWalletData,
+  FilWalletData,
 } from "./abstractWallet";
 import { Client } from "../httpClient";
-import { Balance, Key, Keychains } from "../types";
+import { Balance, Key, PaginationOptions } from "../types";
 import {
   BalanceDTO,
   DepositAddressDTO,
   PatchWalletNameRequest,
 } from "../__generate__/fil";
 import { BNConverter } from "../utils/common";
+import { transformWalletStatus, WalletStatus } from "../wallet";
+import { BlockchainType } from "../blockchain";
+import { FilKeychains } from "./keychains";
 
 export interface FilDepositAddressData
   extends Omit<FilAbstractWalletData, "encryptionKey"> {
   childNumber: number;
 }
 
+export const transformDepositAddressData = (
```

Comment:
btc나 eth 쪽에서는 convertXXX로 쓰고 있는데 통일하면 어떨까요??

Reply:
Actually, in BTC/ETH, `transformXXX` is also used. (eg. `transformMasterWalletData`) :(
How about making a consensus to use `convertXXX` instead of `transformXXX' from now?